src/control/server/config/server_test.go

```diff
@@ -245,6 +246,7 @@ func TestServerConfig_Constructed(t *testing.T) {
 		WithDisableVFIO(true).   // vfio enabled by default
 		WithDisableVMD(true).    // vmd enabled by default
 		WithEnableHotplug(true). // hotplug disabled by default
+		WithClientFirewallMode(true).
```

Comment:
nit: could add similar comment as above for clarity

Reply:
I think we should delete all the comments above instead. The comments didn't add any useful information, lol